### PR TITLE
Cosmetic file changes

### PR DESCRIPTION
**Pull request by cah-hbaum**
This PR tries to update some files in order to provide some cosmetic and logical file changes, like:
- grammar
- wording
- correct linking
- abbreviations
- general cosmetic changes

I already tried to introduce them in earlier updates, but they were rejected due to mix of concerns, so I created this separate PR for them.

### CODE REVIEW

**Review by mbuechse — COMMENTED** [review]
I think I agree with 50 % of this. It's great that some obvious typos are fixed, also some shady syntax is fixed (such as the table headings). I really don't like any changes that are up to taste, such as reformatting whole tables, reflowing paragraphs, changing the meaning of things (from hyperthreads to threads, for example). Finally, I don't like the sheer extent of this PR. It does all these things for all the files all at once. This is still complicated to review and discuss, and I'm not even sure there's demand.

**Comment by cah-hbaum** [comment]
Removed the changes to tables and paragraph reflows.
Didn't find the threads thing you mentioned, but for good measure, I reverted these things.

At last, I won't do this in smaller batches, since I also don't have the time to split this up, but I'm still gonna keep it, since so many mistakes in these documents seems kind of bad for a project of this size.

File: Standards/scs-0400-v1-status-page-create-decision.md [hunk 1]
```diff
@@ -153,7 +153,7 @@ With those requirements in mind the projects that initially were found, were eva
 | user management | ✅ | ❌ | ❌ | ❌ | ✅ by OIDC | ⁇ through github? | ❌ |
 | different output format on notification | ❌ | ❌ | ✅ | ✅ | ✅ | ❌ | ✅ |
 | external hosting | ✅ | ✅ | ✅ | ✅ | ✅ | ❌ looks like you are limited to github | ✅ |
-| project healthy | ❌ last commit 17 months | ❌ last commit 3 years | ❌ last commit 5 months | ✅ last commit 2 months | ✅ recent activities | ✅ recent activities | ❌ archived and abondend by the owner |
+| project healthy | ❌ last commit 17 months | ❌ last commit 3 years | ❌ last commit 5 months | ✅ last commit 2 months | ✅ recent activities | ✅ recent activities | ❌ archived and abndend by the owner |
```
Comment (mbuechse) [inline Standards/scs-0400-v1-status-page-create-decision.md:156]:
```suggestion
| project healthy | ❌ last commit 17 months | ❌ last commit 3 years | ❌ last commit 5 months | ✅ last commit 2 months | ✅ recent activities | ✅ recent activities | ❌ archived and abandoned by the owner |
```

File: Standards/scs-0402-v1-status-page-openapi-spec-decision.md [hunk 2]
```diff
@@ -131,21 +131,21 @@ This means:
...
 
-Generally `POST` requests create new resources. These endpoints do not return the new resource, but a unique identifier to the resource e.g. an UUID.
+Generally `POST` requests create new resources. These endpoints do not return the new resource, but a unique identifier to the resource e.g. a UUID.
 
 In most cases the new resource won't be used directly after creation. Most often list calls are used. If the new resource is used directly, it can be retrieved by the returned identifier.
 
 Payloads to POST requests SHALL NOT include ID or `Incremental` typed fields, it lies in the responsibility of the API server to assign IDs and `Incremental`s to objects.
 
-### Return of `PATCH` requestes
+### Return of `PATCH` requisites
```
Comment (mbuechse) [inline Standards/scs-0402-v1-status-page-openapi-spec-decision.md:148]:
```suggestion
### Return of `PATCH` requests
```

**Review by mbuechse — COMMENTED** [review]
Yes, very good, thanks very much for this! Please do check my comments.

File: Standards/scs-0402-v1-status-page-openapi-spec-decision.md (hunk 2 above)
Comment (cah-hbaum) [reply Standards/scs-0402-v1-status-page-openapi-spec-decision.md]: Updated in https://github.com/SovereignCloudStack/standards/pull/644/commits/50ab7b2c3b9d9274b29b536e69cda56c6c5c0a9b

File: Standards/scs-0400-v1-status-page-create-decision.md (hunk 1 above)
Comment (cah-hbaum) [reply Standards/scs-0400-v1-status-page-create-decision.md]: Updated in https://github.com/SovereignCloudStack/standards/pull/644/commits/50ab7b2c3b9d9274b29b536e69cda56c6c5c0a9b

File: Standards/scs-0213-v1-k8s-nodes-anti-affinity.md [hunk 3]
```diff
@@ -8,7 +8,7 @@ track: KaaS
 ## Introduction
 
 A Kubernetes instance is provided as a cluster, which consists of a set of worker machines,
-so called nodes. A cluster is composed of a control plane and at least one worker node.
+so-called nodes. A cluster is composed of a control plane and at least one worker node.
```
[the remark this answers is not in the document]
Comment (cah-hbaum) [reply Standards/scs-0213-v1-k8s-nodes-anti-affinity.md]: Updated in https://github.com/SovereignCloudStack/standards/pull/644/commits/50ab7b2c3b9d9274b29b536e69cda56c6c5c0a9b

File: Standards/scs-0115-v1-default-rules-for-security-groups.md [hunk 4]
```diff
@@ -18,7 +18,7 @@ Security Group (abbr. SG)
   Set of ip table rules, used for tenant network security.
 
 Security Group Rule (abbr. SG Rule)
-  A single ip table rule, that is part of a SG.
+  A single ip table rule, that is part of an SG.
```
[the remark this answers is not in the document]
Comment (cah-hbaum) [reply Standards/scs-0115-v1-default-rules-for-security-groups.md]: Replaced SG with Security Group.
Updated in https://github.com/SovereignCloudStack/standards/pull/644/commits/50ab7b2c3b9d9274b29b536e69cda56c6c5c0a9b

File: Standards/scs-0114-v1-volume-type-standard.md [hunk 5]
```diff
@@ -93,7 +93,7 @@ openstack volume type show LUKS
 
 ### Replication
 
-Replication states whether or not there are multiple replicas of a volume. Thus, it answers the question whether the data could survive a node outage. Unfortunately there are two ways replication can be achieved:
+Replication states whether or not there are multiple replicas of a volume. Thus, it answers whether the data could survive a node outage. Unfortunately there are two ways replication can be achieved:
```
[the remark this answers is not in the document]
Comment (cah-hbaum) [reply Standards/scs-0114-v1-volume-type-standard.md]: Updated in https://github.com/SovereignCloudStack/standards/pull/644/commits/50ab7b2c3b9d9274b29b536e69cda56c6c5c0a9b

**Review by mbuechse — APPROVED** [review]

**Pull request merged**